updater/script/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+cd "$(dirname "$0")/../.."
+# For simplicity, we just use the bundler base image for the updater environment
+script/build bundler
```

Comment:
I personally am 🤷‍♂️ on whether this adds value... I get that it encapsulates the "use the bundler image as a stand-in" hack, but OTOH, I often find the additional layer of indirection from small scripts like this add more mental overhead than value... and here if I'm looking at this, it's probably because I'm trying to get into the docker dev shell to debug something in the updater, so it's unclear to someone new to this repo how this plays with that.